nsdb-http/src/main/scala/io/radicalbit/nsdb/web/CustomSerializers.scala

```diff
@@ -0,0 +1,191 @@
+ * limitations under the License.
+ */
+
+package io.radicalbit.nsdb.web
+
+import io.radicalbit.nsdb.common.statement._
+import org.json4s.JsonAST.{JDouble, JField, JInt, JLong}
+import org.json4s.{CustomSerializer, JNull, JObject, JString}
+
+object CustomSerializers {
+
+  val customSerializers = List(
+    AggregationSerializer,
+    ComparisonOperatorSerializer,
+    LogicalOperatorSerializer,
+    OrderOperatorSerializer,
+    LikeExpressionSerializer,
+    EqualityExpressionSerializer
+  )
+
+  val customSerializersForTesting = customSerializers ++ List(RelativeComparisonSerializerForTesting)
+
+  case object AggregationSerializer
+      extends CustomSerializer[Aggregation](_ =>
+        ({
+          case JString(aggregation) =>
+            aggregation.toLowerCase match {
+              case "count" => CountAggregation
+              case "max"   => MaxAggregation
+              case "min"   => MinAggregation
+              case "sum"   => SumAggregation
+            }
+          case JNull => null
+        }, {
+          case CountAggregation => JString("count")
+          case MaxAggregation   => JString("max")
+          case MinAggregation   => JString("min")
+          case SumAggregation   => JString("sum")
+        }))
+
+  case object ComparisonOperatorSerializer
+      extends CustomSerializer[ComparisonOperator](_ =>
+        ({
+          case JString(comparison) =>
+            comparison match {
+              case ">"  => GreaterThanOperator
+              case ">=" => GreaterOrEqualToOperator
+              case "<"  => LessThanOperator
+              case "<=" => LessOrEqualToOperator
+            }
+          case JNull => null
+        }, {
+          case GreaterThanOperator      => JString(">")
+          case GreaterOrEqualToOperator => JString(">=")
+          case LessThanOperator         => JString("<")
+          case LessOrEqualToOperator    => JString("<=")
+        }))
+
+  case object LogicalOperatorSerializer
+      extends CustomSerializer[LogicalOperator](_ =>
+        ({
+          case JString(logical) =>
+            logical.toLowerCase match {
+              case "not" => NotOperator
+              case "and" => AndOperator
+              case "or"  => OrOperator
+            }
+          case JNull => null
+        }, {
+          case NotOperator => JString("not")
+          case AndOperator => JString("and")
+          case OrOperator  => JString("or")
+        }))
+
+  case object OrderOperatorSerializer
+      extends CustomSerializer[OrderOperator](_ =>
+        ({
+          case JObject(List(JField("order_by", JString(order)), JField("direction", JString(direction)))) =>
+            direction.toLowerCase match {
+              case "asc"  => AscOrderOperator(order)
+              case "desc" => DescOrderOperator(order)
+            }
+          case JNull => null
+        }, {
+          case AscOrderOperator(order_by) =>
```

Comment:
`orderBy` instead of `order_by`

Reply:
I have to switch to Java/Scala conventions :D

Reply:
exatcly! ahahahah
`order_by` is too python like!